theme/templates/base.html

```diff
@@ -60,11 +60,19 @@ <h1><a href="/">{{ SITENAME }}</a></h1>
   </aside>
 
   <main>
+    <div id="funders">
+      <img id="cbbs" src="{{ SITEURL }}/theme/img/cbbs_logo.png" alt="Logo of CBBS">
```

Comment:
The web UI won't let me comment on the image files, so commenting here. :-/

* CBBS logo, the copy on in the psychoinformatics site repo is already crushed; it's a lot smaller.
* The EFRE combo logo should be crushed (mozjpg or even tinypng.com). Though, if there isn't a strong reason for combining the two, both are also in the psyinf website repo

Reply:
Any reason why you're using ids here? I think ``#funders img`` might be a better fit.

Reply:
Re ids: No valid reason anymore. A remainder of a different approach. Agree on your suggestion.

Re sizes: Agree on using smaller ones, but can't use EFRE from psychoinf site - it's different. Guideline demands to use the combined for Saxony-Anhalt and ERDF and in opposition to what is used on psyinf site it's just the federal state of Saxony-Anhalt, not its ministry for science and economy.